en/publish/publish-cocosplay.md

```diff
@@ -79,6 +73,12 @@ There are two ways to start the game through the **Self-test Tools**.
     - Fill in **path**, such as: <http://192.168.0.1:8080/game.cpk>.
     - Click on **Save** after the configuration is complete, then click on **Start Game** to open the game.
 
+## Resource Management for Cocos Play Environment
+
+**Cocos Play** is similar to **WeChat Mini Game**. There are restrictions on the package size. Resources over **10MB** must be downloaded via a network request.
```

Comment:
Resources over **10MB** must be downloaded via a network request. -> The main package size limit for Cocos Play is **10MB**, more than that must be downloaded via a network request.